lib/lita/state.rb

```diff
@@ -0,0 +1,43 @@
+require_relative "common"
+
+module Lita
+  class State
+    # A handler class that gives in-memory, non-persited, object storage
+    # @since 5.0.0
+    attr_reader :store, :lock
```

Comment:
Either move these readers below the `private` keyword or just use instance variables. This stuff shouldn't be exposed publicly.